app/mci/pkg/gcp/backendservice/backendservicesyncer_test.go

```diff
@@ -39,10 +41,11 @@ func TestEnsureBackendService(t *testing.T) {
 	if _, err := bsp.GetGlobalBackendService(beName); err == nil {
 		t.Fatalf("expected NotFound error, actual: nil")
 	}
-	err := bss.EnsureBackendService(lbName, []ingressbe.ServicePort{
+	beMap, err := bss.EnsureBackendService(lbName, []ingressbe.ServicePort{
 		{
 			Port:     port,
 			Protocol: "HTTP",
+			SvcName:  types.NamespacedName{Name: kubeSvcName},
```

Comment:
Non empty namespace pls?

Reply:
We always omit inconsequential fields in tests. Here namespace does not matter at all. None of the code tested in this unit test reads it.
I can add it if you really want, but that will create the perception that it matters.

Reply:
Got it. Not needed.